lib/tradedesk.js

```diff
@@ -112,7 +112,7 @@ Tradedesk.prototype.__request = function(method, path, params, callback) {
       if (typeof data.errors !== 'undefined') {
         callback(data.errors, data, response);
       }
-      else if(response.statusCode !== 200) {
+      else if(response.statusCode >= 400) {
```

Comment:
Does this library handle 3XX HTTP Codes gracefully? We might want the error to rise when the HTTP code is `>= 300`

Reply:
I think we should be ok.  If TTD returns an error as part of a 3xx response then that would get surfaced as an error in the response from this library due to the if conditional before this.  If there is not an error returned from the tradedesk, then the 3xx would be with the response data.